### DISCUSSION TITLE
Commercial product license

### DISCUSSION THREAD

**Discussion by etwordpress01** [videojs/video.js#7442] [General]
Hello, Hope you are doing great

I have a question related to the license.  Can we use this player in our commercial products for free or do we need to buy a license for commercial use?

We basically building a WordPress plugin and wanted to use this to play some videos. We will sell our plugins on different markets like Envato or Template Monster.

Thank you

**Comment by gkatsev** [comment]
Video.js is licensed under the Apache 2.0 License (https://github.com/videojs/video.js/blob/main/LICENSE), with the full text here:  http://www.apache.org/licenses/LICENSE-2.0. You can do anything you want as long as it's compliant with the license.
